FILE packages/website/src/routes/index.tsx
RANGE {"path":"packages/website/src/routes/index.tsx","old_lines":[74,78],"new_lines":[74,78],"diff_block_lines":[5,11],"hunk_header":"@@ -74,5 +74,5 @@ function Nav() {","substantive_changed_lines":2}
@@ -74,5 +74,5 @@ function Nav() {
         </a>
         <a
-          href="https://github.com/moboudra/paseo"
+          href="https://github.com/getpaseo/paseo"
           target="_blank"
           rel="noopener noreferrer"
